d2.config.js

```diff
@@ -10,9 +10,14 @@ const config = {
             patternsToOmit: [
                 'dashboards/[a-zA-Z0-9]*',
                 'visualizations',
+                'maps',
```

Comment:
Does this exclude the lazy-loaded `maps.js` from the app cache? It might need to be in there

Reply:
Yeah I think this would match /dhis-web-**maps**/map.js, which we want in the app-shell cache because it's not guaranteed to be fetched and recorded when the dashboard is reloaded for recording